src/commands/general/birthday/test.ts

```diff
@@ -1,21 +1,24 @@
+import { defaultClientPermissions, defaultUserPermissions } from '#lib/types';
+import { interactionSuccess } from '#utils/embed';
+import { reply, resolveTarget } from '#utils/utils';
 import { Command, RegisterSubCommand } from '@kaname-png/plugin-subcommands-advanced';
-import { reply } from '../../../helpers';
-import { interactionSuccess } from '../../../lib/utils/embed';
-import { testBirthdaySubCommand } from '../../../lib/commands';
-import { RequiresUserPermissions } from '@sapphire/decorators';
+import { RequiresClientPermissions, RequiresGuildContext, RequiresUserPermissions } from '@sapphire/decorators';
+import { testBirthdaySubCommand } from './birthday';
 
 @RegisterSubCommand('birthday', (builder) => testBirthdaySubCommand(builder))
 export class TestCommand extends Command {
-	@RequiresUserPermissions('ManageGuild')
+	@RequiresGuildContext()
+	@RequiresClientPermissions(defaultClientPermissions)
+	@RequiresUserPermissions(defaultUserPermissions)
```

Comment:
the user needs the `manage roles` or `manage server` to be able to use the `/birthday test` command.

Reply:
ah are the permission levels for this?